edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/AuthorizationConfiguration.cs

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Core.Config
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Domain object that represents Authorization configuration for Edge Hub Module (MQTT Broker).
+    ///
+    /// This object is being eventually constructed from the EdgeHub twin's desired properties.
+    /// See <see cref="EdgeHubDesiredProperties"/> for DTO.
+    /// </summary>
+    public class AuthorizationConfiguration : List<Statement>, IEquatable<AuthorizationConfiguration>
```

Comment:
I would add a list of statements instead of extending List<Statement>

Reply:
Extending a list allows me to define my own deep equality comparison logic without extra level of indirection (a class that wraps a list). I guess rust has a "bad" influence on me :D. I thought about that and would like to leave it as is, unless you have strong reasons not to do that.